draft-ietf-rats-corim.md

```diff
@@ -219,6 +219,9 @@ Class ID:
 : An identifier for an Environment that is shared among similar Environment instances, such as those with the same hardware assembly.
 See also {{Section 4.2.4 of -eat}}.
 
+Domain:
+: A topological description to represent an entity, for example a Composite Device ({{Section 3.3 of -rats-arch}}), in terms of its constituent Environments or Domains.
```

Comment:
Defining Domain in terms of Domains is like looking in an infinity mirror :-)

Reply:
Changes done!